### DISCUSSION TITLE
What is the eye icon used for?

### DISCUSSION THREAD

**Discussion by Asky740** [DanielRendox/RoutineTracker#27] [Q&A] [answered]
Hey, just found out about this app and it's really awesome. But I can't figure out what is the eye icon on the main screen used for. Thanks.

**Comment by DanielRendox** [comment, marked as answer]
Hey, this icon shows habits that are not due on a given date.

For example, if you create a habit that repeats every Tuesday, it will be
hidden on Monday. However, if you want to complete it anyway, you can
toggle the eye icon to show it.

It's a bit confusing, so I'll probably add another screen separate from the
agenda where all habits will be displayed. Then this feature won't be
needed anymore.
